presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -620,16 +625,33 @@ public void registerTableForView(Table tableReference)
         tablesForView.push(requireNonNull(tableReference, "table is null"));
     }
 
+    public void registerTableForMV(Table table)
```

Comment:
For all the word "MV" in the PR, we will need to replace it with "MaterializedView". In our codebase, we don't use abbreviations.